inc/saisamplepacket.h

```diff
@@ -41,6 +41,9 @@ typedef enum _sai_samplepacket_type_t
     /** Copy the sample packets to CPU */
     SAI_SAMPLEPACKET_TYPE_SLOW_PATH,
 
+    /** Copy the sample packets using mirror session */
+    SAI_SAMPLEPACKET_TYPE_MIRROR_SESSION,
```

Comment:
From reference guide I read, some chip is capable of copying a packet sample to CPU and, at the same time, sending the sample to 1 or more mirror sessions . Not sure of any practical use case of such a capability, but as `SAI_PORT_ATTR_INGRESS_SAMPLEPACKET_ENABLE` and `SAI_PORT_ATTR_EGRESS_SAMPLEPACKET_ENABLE` takes only one sample object id, not an objlist like the case of `SAI_PORT_ATTR_INGRESS_MIRROR_SESSION` and `SAI_PORT_ATTR_EGRESS_MIRROR_SESSION`, this limits a sample to be either sent to CPU or sent to mirror session(s) as stated in the first sentence of the proposal, but not both as some chip can be capable of.

Reply:
@wendani Currently, having a single sample object OID will help us configure the same sampling rate for all mirror sessions (which was the proposal) and making it a list will over complicate it. I would like to keep it as is and would like to take this proposal forward.